nosetimer/plugin.py

```diff
@@ -1,8 +1,8 @@
+# -*- mode: python; coding: utf-8 -*-
```

Comment:
Do we really need this?

Reply:
No, not really. Its a good practice in projects i work with. And reflects PEP 263.
I can remove it.